### PR TITLE
Update replication.html

### PR DESCRIPTION
**Pull request by PatrickHeneise**
Took me 10 minutes to figure out that CouchDB isn't listening when doing a remote replication. This should be mentioned in the docs.

### CODE REVIEW

**Comment by janl** [comment]
Heya Patrick, thanks for the addition. This isn’t 100% accurate, as only one of the two couches needs to be on a public IP. The one you are working on can be all private and bound to 127.0.0.1 _if_ the other couch is publicly available. Could you rephrase the paragraph to reflect this? Otherwise, this is a great addition!

**Comment by PatrickHeneise** [comment]
Rephrased it to open the remote server only. For continuous replication, do you need both servers to be public?

**Comment by janl** [comment]
@PatrickHeneise `continuous` is just an option of replication, same rules as for non-continuous replications apply.

Thanks!
